src/Integrations/Integrations/Symfony/SymfonyIntegration.php

```diff
@@ -248,46 +248,36 @@ function ($This, $scope, $args) {
             }
         );
 
-        $symfonyCommandsIntegrated = [];
-        \DDTrace\hook_method(
+        \DDTrace\trace_method(
             'Symfony\Component\Console\Command\Command',
-            '__construct',
-            null,
-            function ($This, $scope) use (&$symfonyCommandsIntegrated, $integration) {
-                if (isset($symfonyCommandsIntegrated[$scope])) {
-                    return;
-                }
-
-                $symfonyCommandsIntegrated[$scope] = true;
-
-                \DDTrace\trace_method($scope, 'run', [
```

Comment:
I assume this was an artifact of pre-historical times when we had no support for abstract method hooks?

At a glance this fix looks pretty good to me.